DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_CompetencySummary.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@using DigitalLearningSolutions.Data.Models.SelfAssessments
+@{
+  var selfAssessment = (CurrentSelfAssessment)ViewData["selfAssessment"];
+  var questions = ((IGrouping<string, Competency>)ViewData["competencyGroup"])
+    .SelectMany(c => c.AssessmentQuestions)
+    .Where(q => q.Required);
+  var selfAssessed = questions.Count(q => q.Verified == null && q.ResultId.HasValue);

```

Comment:
This wants to be `var selfAssessed = questions.Count(q => q.ResultId.HasValue);`
We should include verified results in the self assessed count.